templates/login.html

```diff
@@ -0,0 +1,36 @@
+{{ define "content" }}
+
+<h2 class="auth-main__title">
+  Login to your account
+</h2>
+
+{{ if .notices }}
+  <div class="alert alert--success">
+    {{ range $notice := .notices }}
+      {{ $notice }}
+    {{ end }}
+  </div>
+{{ end }}
+
+{{ if .errors }}
+  <div class="alert alert--danger">
+    {{ .errors }}
+  </div>
+{{ end }}
+
+
```

Comment:
One extra blank line can be removed

Reply:
Could you please check this again? I didn't see any extra blank line

Reply:
Looks fine now 😁